ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,26 @@
+import {useState} from "react"
+
+export default function SortCursor(props){
+
+    const[currentClickedCursor,setcurrentClickedCursor] = useState('')
+
+    const onClickUpArrow = () =>{
+        setcurrentClickedCursor("UP")
```

Comment:
Rename: `setcurrentClickedCursor` -> `setCurrentClickedCursor`
move `"UP" `to constant